test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py

```diff
@@ -189,6 +196,8 @@ def _check_version(self, node, version, collection_name):
                 if collection_name == self.collection_name and self.collection_version is not None:
                     if self.collection_version >= semantic_version:
                         self.add_message('collection-deprecated-version', node=node, args=(version,))
+                if semantic_version.major != 0 and (semantic_version.minor != 0 or semantic_version.patch != 0):
```

Comment:
```suggestion
                if semantic_version.major == 0 or (semantic_version.major != 0 and (semantic_version.minor != 0 or semantic_version.patch != 0)):
```

Reply:
0.x.y as a removal version is fine, in 0.x.y versions anything can be done (see https://semver.org/ point 4 in the spec).

Reply:
ok